apps/prairielearn/src/server.js

```diff
@@ -175,15 +174,6 @@ export async function initExpress() {
   // API routes don't utilize sessions; don't run the session/flash middleware for them.
   app.use(excludeRoutes(['/pl/api'], sessionRouter));
 
-  app.use(function (req, res, next) {
-    if (req.headers['user-agent']) {
-      res.locals.userAgent = Bowser.parse(req.headers['user-agent']);
-    } else {
-      res.locals.userAgent = null;
```

Comment:
AFAICT this is the last time this stopped being used: https://github.com/PrairieLearn/PrairieLearn/pull/6974/files#diff-cae3cb6cc3f271a4dd1c43d90e822ceb3d82a56884ffa57728af052ed0aaa284